context.go

```diff
@@ -151,17 +151,21 @@ func (c *Context) setDefaultTags() error {
 	return nil
 }
 
-// Stats gets stats for a context as a string
-func (c *Context) Stats() (string, error) {
+// Stats gets stats for a context as json
+func (c *Context) Stats() ([]byte, error) {
 	var stats *C.char
 	if ret := C.tiledb_ctx_get_stats(c.tiledbContext, &stats); ret != C.TILEDB_OK {
-		return "", fmt.Errorf("Error getting stats from context: %s", c.LastError())
+		return nil, fmt.Errorf("Error getting stats from context: %s", c.LastError())
 	}
 
 	s := C.GoString(stats)
 	if ret := C.tiledb_stats_free_str(&stats); ret != C.TILEDB_OK {
-		return "", fmt.Errorf("Error freeing string from dumping stats to string")
+		return nil, fmt.Errorf("Error freeing string from dumping stats to string")
 	}
 
-	return s, nil
+	if s == "" {
+		return []byte("{}"), nil
```

Comment:
should this be `{}` or `null`? (either might be fine, just want to make sure we're doing the right thing here)

Reply:
I think either is okay, I think null didn't use to be valid according to json spec but it looks like this has changed